pkg/conversion/converter.go

```diff
@@ -495,15 +495,6 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
 // one is registered.
 func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
 	dt, st := dv.Type(), sv.Type()
-	// Apply default values.
-	if fv, ok := c.defaultingFuncs[st]; ok {
```

Comment:
Wow, this is a big change. I mean yes they are getting called twice, but are there manual conversion functions which don't call the autogenerated ones and also don't call defaulting?